src/ModbusSlave.h

```diff
@@ -100,12 +115,14 @@ class Modbus {
     uint64_t getTotalBytesSent();
     uint64_t getTotalBytesReceived();
 
-    MobbusCallback cbVector[CB_MAX];
+    ModbusCallback* cbVector;
```

Comment:
Can we remove this `cbVector` now ?

Reply:
Can you add a comment ? something like: 
```
// This cbVector is a pointer to cbVector of the first slave, to allow shorthand syntax:
//     Modbus slave(SLAVE_ID, CTRL_PIN);
//     slave.cbVector[CB_WRITE_COILS] = writeDigitalOut;
// Instead of the compleate:
//     ModbusSlave slaves[1] = { ModbusSlave(ID_SLAVE_1) };
//     Modbus modbus(slaves, 1);
//     slaves[0].cbVector[CB_WRITE_COILS] = writeDigitalOut;
```